kong/plugins/aws-lambda/handler.lua

```diff
@@ -63,6 +63,28 @@ local function get_now()
 end
 
 
+local function validate_http_status_code(status_code)
+  if not status_code then
+    return false
+  end
+
+  if type(status_code) == "string" then
+    status_code = tonumber(status_code)
+
+    if not status_code then
+      return false
+    end
+
```

Comment:
```suggestion
```

We don't need a newline if there are no more code after the block.

Reply:
Got it, thanks!